Automattic/woocommerce-payments
# Adding manual option to deposit schedule selection

Fixes #4696

#### Changes proposed in this Pull Request

Introduce top-level radio selections, `Automatic` and `Manual`, for deposits schedule.

#### Testing instructions

* Run `npm run build:client`.
* Go to wp-admin > Payments > Settings.
* Make sure `Automatic` and `Manual` radios appear below `Deposit schedule`.
* Make sure they are functional, eg selecting `Manual` and clicking the `Save changes` button will set the Stripe account payout schedule to `Manual` (check the Stripe dashboard to confirm this).
* The deposit settings match the designs → paJDYF-6a4-p2


-------------------


**Post merge**
